azblob/version.go

```diff
@@ -1,3 +1,3 @@
 package azblob
 
-const serviceLibVersion = "0.7"
+const serviceLibVersion = "0.7.1"
```

Comment:
There's no need to update the version yet, as we usually do that right before releasing.